README.md

```diff
@@ -553,6 +609,7 @@ All requests use the same authentication methods as the upload call (Basic Authe
 ### Known Issues
 
 - Node will show compilation warnings when using Raygun4Node in Webpack applications.
+- Although Breadcrumbs report the source filename, code line and function name, these are not processed using source maps like with stack traces.
```

Comment:
Do we plan to tackle this in an upcoming release?

Reply:
No, this is not a client issue, but a general problem with Raygun that should be implemented on the server side. I thought about mentioning this here since the topic of source mapping and breadcrumbs came up in a comment.